packages/website/docusaurus.config.ts

```diff
@@ -80,12 +80,12 @@ const config: Config = {
   },
   scripts: [
     {
-      src: "es-module-shims.js",
+      src: "/es-module-shims.js",
```

Comment:
I might have to use the computed `baseUrl` instead of `/` so it works in the Pr try it, we'll have to see